package.json

```diff
@@ -42,13 +42,13 @@
     "copy-webpack-plugin": "^5.0.3",
     "css-loader": "^3.0.0",
     "designsystemet": "git+https://github.com/Informasjonsforvaltning/designsystemet.git#1.4.24",
-    "dotenv": "^8.0.0",
     "ejs": "^2.6.1",
     "express": "^4.17.0",
     "fast-json-patch": "^3.0.0-0",
     "file-loader": "^4.0.0",
     "font-awesome": "^4.7.0",
     "formik": "^1.5.7",
+    "html-webpack-plugin": "^3.2.0",
```

Comment:
This belongs to devDependencies

Reply:
vi kan flytte det senere der.
akkurat nå er det semantics, at "alt som trenges i build" er prod dependencies (se Dockerfile, vi bruker `RUN npm install --only=production --loglevel=warn`).
Senere, når vi kjører test i konteiner, og har multistage build, kan vi ha full npm install, og mer semantisk betydning i devDependencies